### DISCUSSION TITLE
list_members() not returning all members

### DISCUSSION THREAD

**Discussion by MoHayat** [tweepy/tweepy#1188] [Questions] [answered]
Hello, 

I tried using the list_members() function to retrieve 87 users from a specific list, but all I got were the first 20 users. I tried to see whether the full list was split into pages but it wasn't. Is that a mistake on my end or is there some issue with obtaining the data?

Thanks for the help.

**Comment by Harmon758** [comment, marked as answer]
I'm unable to reproduce this.
Are you sure you're not only getting the first page? Have you tried specifying the `count` parameter?
The default is 20, so that's how many you would get from the first page without specifying it.
The alternative is to retrieve each page of results, e.g. [via a `Cursor`](http://docs.tweepy.org/en/latest/cursor_tutorial.html).
Also note, "Private list members will only be shown if the authenticated user owns the specified list."
See https://developer.twitter.com/en/docs/accounts-and-users/create-manage-lists/api-reference/get-lists-members

**Comment by MoHayat** [comment]
I see, I was able to get the full list using a Cursor.

 thanks for the help!
